recursio-client/Managers/CharacterManager.gd

```diff
@@ -175,13 +141,20 @@ func _on_countdown_phase_started() -> void:
 	
 
 func _on_game_phase_started() -> void:
+	_player.set_record_data_timestamp(Server.get_server_time())
+	_enemy.set_record_data_timestamp(Server.get_server_time())
 	_player.block_movement = false
+	_enemy.block_movement = false
 	_player.set_overview_light_enabled(false)
 	_toggle_visbility_lights(true)
 	_game_manager.hide_countdown_screen()
 	_player.show_game_hud(_round_manager.round_index)
 	_game_manager.toggle_capture_points(true)
 
+func _on_game_phase_stopped() -> void:
+	_game_manager.toggle_capture_points(false)
```

Comment:
Why are we toggling those off?

Reply:
We always enable then on game phase start (line 152) but never disable them. I also had some problems with them being on for a split second when the game phase is over

Reply:
yeah sure but do we want to disable them? I think they should be visible during the prep phase, so the player can plan his round better? Or am I misunderstanding what `toggle_capture_points `does.

Reply:
It just disables the process and thereby doesn't allow the capturing process to be changed. Furthermore should the process be reset to 0 on game_phase_end

Reply:
Oh okay I thought this would hide the mesh. Nevermind then, sorry.